addons/point_of_sale/static/src/js/models.js

```diff
@@ -2445,11 +2451,12 @@ exports.Order = Backbone.Model.extend({
             orderlines: orderlines,
             paymentlines: paymentlines,
             subtotal: this.get_subtotal(),
-            total_with_tax: this.get_total_with_tax(),
+            total_with_tax: this.get_total_with_tax() + this.get_rounding_applied(),
```

Comment:
you should show both, rounded and not rounded amount on the ticket. So I thinkn you only have the rounded total now, and not the normal total anymore.

Reply:
The "real" total is shown as the subtotal on the receipt.
The total shown there is the total including the rounded value, which is what the customer will pay.
Maybe this is confusing... Should i rename those values?